### ISSUE TITLE
test-helpers.js file from blueprint references non-existing file

### ISSUE THREAD

**Issue by simonihmig** [ember-cli/ember-cli-mocha#231]
As discussed on Slack, the blueprint file references `./helpers/resolver` [here](https://github.com/ember-cli/ember-cli-mocha/blob/master/blueprints/ember-cli-mocha/files/tests/test-helper.js#L1), which has been removed recently from ember-cli's app blueprint [here](https://github.com/ember-cli/ember-cli/commit/4d21f92ddf2b25cbae2d9326d7b83125bb8e3173#diff-d4cde49921e37c6199d46a7bff66f061), causing a `Could not find module ...` error when running tests.

**Comment by Turbo87** [comment]
I guess the best solution for now is to copy the removed resolver into our own blueprint.

For the current beta release based on https://github.com/ember-cli/ember-cli-mocha/pull/229 we need to update the `test-helper` blueprint instead to use the `setApplication()` function too.

**Comment by mattmarcum** [comment]
Looks like `resolver.js` was removed in ember-cli@2.17 - here it is in 2.16: https://github.com/ember-cli/ember-cli/blob/v2.16.0/blueprints/app/files/tests/helpers/resolver.js

```js

import Resolver from '../../resolver';
import config from '../../config/environment';

const resolver = Resolver.create();

resolver.namespace = {
  modulePrefix: config.modulePrefix,
  podModulePrefix: config.podModulePrefix
};

export default resolver;
```

**Comment by Turbo87** [comment]
Resolved by #237
